features/step_definitions/authorization.js

```diff
@@ -37,4 +46,82 @@ module.exports = function server() {
             return null;
         });
     });
+
+    this.Then(/^they can see the project$/, () =>
+        request({                           // make sure pipeline exists
```

Comment:
> make sure the pipeline exists

Wouldn't this be better placed in the *Given an existing pipeline with that repository* step definition?